echopype/convert/api.py

```diff
@@ -73,14 +74,19 @@ def _validate_path(
         out_path = out_dir / (Path(source_file).stem + file_format)
 
     else:
-        fsmap = fsspec.get_mapper(save_path, **output_storage_options)
+        if isinstance(save_path, str):
+            save_path = Path(save_path)
```

Comment:
I think the issue is here. Using pathlib Path removes the `://` and turns it to `:/`.

![Screenshot from 2021-04-26 12-03-18](https://user-images.githubusercontent.com/17802172/116136642-69c13e80-a687-11eb-92b3-cc5a9cfff3c3.png)

Reply:
Yes, Path `doesn't` handle url's, last time I checked. So you have to test for Path vs (http://, https://) vs s3:// (the last two sets as strings), and make decisions accordingly.